share/picongpu/tests/Synchrotron_plugin/lib/change_parameters_randomly.py

```diff
@@ -0,0 +1,55 @@
+import os
+import numpy as np
+from synchrotron_lib import calculate_dt, predictNumPhotons
+# dt is in grid.param as DELTA_T_SI
+# Heff is in fieldBackground.param as field_Strength_SI
+# gamma is in particle.param as gamma
+# all are in ./include/picongpu/param
+
+paramsPath = os.path.join(os.path.dirname(os.path.realpath(__file__)), "../include/picongpu/param")
```

Comment:
In modern Python, one should prefer `pathlib`, roughly (please check the details):
```suggestion
paramsPath = Path(__file__).absolute().parent /  "../include/picongpu/param"
```

Reply:
still open - might be a thing for a future refactoring

Reply:
Yes, that's totally fine.